lib/src/repository.rs

```diff
@@ -27,8 +27,40 @@ pub trait RepositoryExt {
     /// For a given message of an issue, find the initial message.
     ///
     fn find_tree_init<'a>(&'a self, commit: Commit<'a>) -> Result<Commit>;
+
+    /// Get issue hashes for a prefix
+    ///
+    /// This function returns all known issues known to the DIT repo under the
+    /// prefix provided (e.g. all issues for which refs exist under
+    /// `<prefix>/dit/`). Provide "refs" as the prefix to get only local issues.
+    ///
+    fn get_issue_hashes(&self, prefix: &str) -> Result<Vec<Oid>>;
+
+    /// Get all issue hashes
+    ///
+    /// This function returns all known issues known to the DIT repo.
+    ///
+    fn get_all_issue_hashes(&self) -> Result<Vec<Oid>>;
```

Comment:
An unsized trait cannot be wrapped in a result or returned directly and I didn't feel it would be the right way to explicitly write out the type. Or does Rust have something similar to the `auto` keyword on C++?